src/test/java/org/openmrs/module/mirebalais/smoke/PatientRegistrationLiberiaFlowTest.java

```diff
@@ -1,20 +1,73 @@
 package org.openmrs.module.mirebalais.smoke;
 
 import org.junit.Test;
+import org.openmrs.module.mirebalais.smoke.dataModel.Patient;
+import org.openmrs.module.mirebalais.smoke.helper.PatientDatabaseHandler;
+import org.openmrs.module.mirebalais.smoke.pageobjects.ClinicianDashboard;
+import org.openmrs.module.mirebalais.smoke.pageobjects.PatientRegistration;
 import org.openqa.selenium.By;
 
+import java.util.Random;
+
+import static org.junit.Assert.assertTrue;
+
 public class PatientRegistrationLiberiaFlowTest extends PatientRegistrationFlowTest {
 
     @Test
     @Override
     public void registerNewPatient() throws Exception {
-        super.registerNewPatient();
+        initBasicPageObjects();
+        setLoginPage(getLoginPage());
+        PatientRegistration registration = new PatientRegistration(driver);
+
+        String givenName = "Tom " + new Random().nextInt(1000);  // append a random number so patient name is (more or less) unique
+        String familyName = "Jones";
+        String nickname = "Tommy";
+
+        login();
+        appDashboard.openPatientRegistrationApp();
+        //click on the Register Patient button
+        driver.findElement(By.id("register-patient-button")).click();
+
+        registration.registerPatient(givenName, familyName, nickname, PatientRegistration.Gender.MALE, 22, 4, 1975, "louise", getPlaceOfBirthString(),
+                getPersonAddressString(), getContactPhoneNumber(), getInsuranceName(),getInsuranceNumberString(), getOtherInsuranceNameString(), getMaritalStatus(), getOccupation(), getReligion(),
+                getContact(), getRelationship(), getContactAddressString(), placeOfBirthAndContactAddressUseHierarchy(), "0885778899", automaticallyEnterIdentifier(), getRelationshipsEnabled(), getBiometricsEnabled(), getPrintIdCardOption(),
+                getAdditionalIdentifiersEnabled(), getSuccessElement());
+
+        appDashboard.goToAppDashboard();
+        appDashboard.findPatientByGivenAndFamilyName(givenName, familyName);
+        assertTrue(new ClinicianDashboard(driver).isOpenForPatient(givenName, familyName));
+
+        populateTestPatientForTearDown();
+        logout();
```

Comment:
@alaboso why are we replicating this code?  What changes here? 

(maybe I'm just missing it and there's a good reason, let me know!)